soccer/Processor.cpp

```diff
@@ -84,7 +84,8 @@ Processor::Processor(bool sim, bool defendPlus, VisionChannel visionChannel,
     _vision = std::make_shared<VisionFilter>();
     _refereeModule = std::make_shared<NewRefereeModule>(&_context);
     _refereeModule->start();
-    _gameplayModule = std::make_shared<Gameplay::GameplayModule>(&_context);
+    _gameplayModule = std::make_shared<Gameplay::GameplayModule>(
+        &_context, _refereeModule.get(), &_grCom);
```

Comment:
I still don't love this. IMO it would be significantly preferable to pass around in `Context` a `optional<BallMovementRequest>` that gets reset to `nullopt` by the GrSim communicator after it has been serviced.

Come to think of it, doing the same with a `GameStateOverrideRequest` might solve some of the python nastiness. I'll need to think about this more, because on one hand it's basically baking an RPC framework into our codebase, but to be fair once we're moving over to ROS that's what we'll be using anyway.